Pages/signup/signup.vue.js

```diff
@@ -0,0 +1,109 @@
+const signUpComponent = {
+    data: function() {
+        return {
+            user: {
+                firstName : '',
+                lastName : '',
+                email : '',
+                password : '',
+                confirmPassword : '',
+                phonenumber : ''
+            }
+        }
+    },
+    methods: {
+        createUser: function() {
+            db.collection('user').add({
+                firstName: this.user.firstName,
+                lastName: this.user.lastName,
+                email: this.user.email,
+                phonenumber: this.user.phonenumber
+            }).then(auth.createUserWithEmailAndPassword(this.user.email, this.user.password).then(() => {
+                auth.signInWithEmailAndPassword(this.user.email, this.user.password).then(() => {
+                    auth.currentUser.updateProfile({displayName: this.user.firstName + ' ' + this.user.lastName})
+                    auth.currentUser.sendEmailVerification().then(auth.signOut())
+                }
+                ).catch(function(error) {
+                    // Handle Errors here.
+                    var errorCode = error.code;
+                    var errorMessage = error.message;
+                    // ...
+                    console.log("error code", errorCode)
+                    console.log("error Message", errorMessage)
+                  });
+                
+            }))
+            // auth.signOut()
+            this.$router.push('/')
```

Comment:
I think its good to put TODO: here, something like _update the route to signIn once that functionality implemented_